crash_similarity.py

```diff
@@ -15,9 +15,33 @@
 import numpy as np
 import bisect
 import pyximport; pyximport.install()
+from datetime import datetime
 
 # import download_data
 
+# Checks if the model has been trained in the last 24 hours
+def checkif_model_trained_in_last_24hours():
+    with open('last_trained.txt', 'r') as myfile:
+        data = myfile.read()
+        last_trained_time = datetime.strptime(data, '%b %d %Y %I:%M%p')
+        last_trained_time_value = last_trained_time.day*86400 + last_trained_time.second
+
+    cur_time = datetime.today()
+    cur_time_value = cur_time.day*86400 + cur_time.second
+
+    str = cur_time.strftime('%b %d %Y %I:%M%p')
+
+    with open("last_trained.txt", "w") as text_file:
+
+        text_file.write(str + "\n")
+
+    time_diff = cur_time_value - last_trained_time_value
+
+    if(time_diff>86400):
```

Comment:
why not `return time_diff>86400` ?
don't forget about PEP8: you should add spaces before and after `>`
Anyway, it is better to rewrite this method completely, using python datetime library.